bitnami/rabbitmq/values.yaml

```diff
@@ -97,11 +97,12 @@ rabbitmq:
   setUlimitNofiles: true
   ulimitNofiles: '65536'
 
-  ## RabbitMQ maximum available scheduler threads and online scheduler threads
+  ## RabbitMQ maximum available scheduler threads and online scheduler threads. By default it will create a thread per CPU detected, with the following parameters you can tune it manually.
   ## ref: https://hamidreza-s.github.io/erlang/scheduling/real-time/preemptive/migration/2016/02/09/erlang-scheduler-details.html#scheduler-threads
+  ## ref: https://github.com/bitnami/charts/issues/2189
   ##
-  maxAvailableSchedulers: 2
-  onlineSchedulers: 1
+  # maxAvailableSchedulers: 2
+  # onlineSchedulers: 1
```

Comment:
Do you know which could be the implications of remove this configuration if you had the chart installed from the previous version setting this? Could this change break it?
For example, if a user installed the version `6.24.0` those parameters will have a value, then, if they upgrade the chart to this version the value of those paramters will be removed.

Reply:
The implications will be that as the Erlang VM is setting this it will choose an appropiate number of threads for the CPUs detected. Maybe you get an increase in performance.

Reply:
I will trust you